Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProjectsPage.java

```diff
@@ -181,23 +181,29 @@ public int countListedRulesets() throws Exception {
     }
 
     public void createNewProcess() {
+        int index = triggerRowToggle(templatesTable, "First template");
+        WebElement createProcess = Browser.getDriver()
+                .findElement(By.id(TEMPLATE_TABLE + ":" + index + ":createProcessForm:projects:0:createProcess"));
         createProcess.click();
     }
 
     public List<String> getProjectDetails() {
         int index = triggerRowToggle(projectsTable, "First project");
         WebElement detailsTable = Browser.getDriver()
-                .findElement(By.id("projectsTabView:projectsTable:" + index + ":projectDetailTable"));
+                .findElement(By.id(PROJECTS_TABLE + ":" + index + ":projectDetailTable"));
         return getTableDataByColumn(detailsTable, 1);
     }
 
-    public List<String> getTemplateDetails() {
+    public List<String> getTemplateDetails() throws Exception {
         int index = triggerRowToggle(templatesTable, "First template");
+        Thread.sleep(5000);
```

Comment:
Why did you add this and a few lines below this call and only returning an empty list? I think this and similar line below could be removed.